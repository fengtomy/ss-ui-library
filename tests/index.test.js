import pkg from '../package.json';

const testDist = process.env.LIB_DIR === 'dist';

describe('ss-ui-library dist files', () => {
  // https://github.com/ant-design/ant-design/issues/1638
  // https://github.com/ant-design/ant-design/issues/1968
  it('exports modules correctly', () => {
    const rcantd = testDist ? require('../dist/ss-ui-library') : require('../components'); // eslint-disable-line
    expect(Object.keys(rcantd)).toMatchSnapshot();
  });

  // https://github.com/ant-design/ant-design/issues/1970
  // https://github.com/ant-design/ant-design/issues/1804
  if (testDist) {
    it('should have ss-ui-library.version', () => {
      const rcantd = require('../dist/ss-ui-library'); // eslint-disable-line
      expect(rcantd.version).toBe(pkg.version);
    });
  }
});
