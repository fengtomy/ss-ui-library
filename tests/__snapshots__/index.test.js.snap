// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ss-ui-library dist files exports modules correctly 1`] = `
Array [
  "Star",
  "version",
]
`;
